static/js/popovers.js

```diff
@@ -501,7 +501,11 @@ exports.register_click_handlers = function () {
         e.preventDefault();
     });
     $('body').on('click', '.respond_personal_button', function (e) {
-        compose_actions.respond_to_message({reply_type: 'personal', trigger: 'popover respond pm'});
+        var user_id = $(e.target).parents('ul').attr('data-user-id');
+        var email = people.get_person_from_user_id(user_id).email;
+        compose_actions.start('private', {
+            trigger: 'popover respond personal button',
```

Comment:
I think this should probably be changed to `popover send private`, given that's a fair description of the user case.

Otherwise lgtm; I would normally ask for unit tests, but I think we don't have any for this subsystem.

Reply:
Fixed.